Makefile

```diff
@@ -124,7 +128,7 @@ sanic:
 # --- Benchmarker ---
 # client
 client:
-	cd benchmarker; crystal build src/client.cr -o bin/client --release
+	cd benchmarker; crystal build src/client.cr -o bin/client --release --link-flags -L/usr/local/opt/openssl/lib
```

Comment:
I don't think this is necessary for all users. I can build this without the option on my Mac. Could you remove the line?